src/components/ArticleNavigations/ArticleNavigations.tsx

```diff
@@ -0,0 +1,52 @@
+import {ArrowLeft, ArrowRight} from '@gravity-ui/icons';
+import {Button, Flex, Icon, Text} from '@gravity-ui/uikit';
+import {block} from 'src/utils';
+
+const b = block('article-navigations');
+
+import './ArticleNavigations.scss';
+
+interface ArticleNavigationsProps {
+    previousTitle: string;
+    nextTitle: string;
+    nextHandler: () => void;
+    prevHandler: () => void;
+}
+
+export const ArticleNavigations = ({
+    previousTitle,
+    nextTitle,
+    nextHandler,
+    prevHandler,
+}: ArticleNavigationsProps) => {
+    return (
+        <div className={b()}>
+            <div className={b('item')}>
+                <Button onClick={prevHandler} className={b('item-button')} width="max">
+                    <Icon data={ArrowLeft} size={16} />
+                </Button>
+                <Flex direction="column" gap="1" style={{padding: '8px 0'}}>
```

Comment:
if u want use inline styles for paddings - correct way is use `spacing utility`
https://preview.gravity-ui.com/uikit/?path=/docs/layout--docs#spacing-utility

className={spacing({py: 2})}